resources/migrate-to-swift.md

```diff
@@ -181,4 +182,4 @@ Finally, you'll need to scan through your current implementation and replace ACP
 | [Adobe Audience](../using-mobile-extensions/adobe-audience-manager/) | [AEPAudience](../using-mobile-extensions/adobe-audience-manager/migration.md) |
 | [Adobe Experience Platform Target](../using-mobile-extensions/adobe-target/) | [AEPTarget](../using-mobile-extensions/adobe-target/migration.md) |
 | [Adobe Experience Platform Campaign](../using-mobile-extensions/adobe-campaign-standard/) | [AEPCampaign](../using-mobile-extensions/adobe-campaign-standard/migration.md) |
-
+| [Adobe Experience Platform Campaign Classic](../using-mobile-extensions/adobe-campaignclassic/) | [AEPCampaignClassic](../using-mobile-extensions/adobe-campaignclassic/migration.md) |
```

Comment:
Would like to keep this change in a separate PR. 
We would need a grep on the whole repo to find out the references to these pages.

Reply:
Since this stuff is migrating over to the dev.adobe by October, I don't think we should put too much effort in grepping the repo to rename, since it'll be changing soon.